packages/fluentui/accessibility/src/behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -34,11 +34,6 @@ const treeTitleBehavior: Accessibility<TreeTitleBehaviorProps> = props => ({
       performClick: {
         keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
       },
-      ...(props.selectable && {
```

Comment:
that is good change, just please remove as well description above :) and probably unit tests will fails, when you removed the code :)

Reply:
Done